core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -128,10 +128,9 @@ class TransactionMarkerChannelManager(config: KafkaConfig,
 
   private val markersQueuePerBroker: concurrent.Map[Int, TxnMarkerQueue] = concurrent.TrieMap.empty[Int, TxnMarkerQueue]
 
-  private val interBrokerListenerName: ListenerName = config.interBrokerListenerName
+  private val markersQueueForUnknownBroker: TxnMarkerQueue = new TxnMarkerQueue(Node.noNode)
```

Comment:
Intellij IDE states the opposite :) Anyways, I know Ismael is a strong advocator against those redundants so I will change it back.